File: App/PixelArtHunter/app/src/main/java/net/kteam/pixelarthunter/POI.kt
package net.kteam.pixelarthunter

class POI(
    val id: Int,
    val name: String,
    var longitude: Double,
    var latitude: Double,
    var modifier: Double
) {
    // Custom constructor
    constructor(id: Int, name: String, longitude: Double, latitude: Double) : this(id, name, longitude, latitude, 1.0)
}

File: Server/app/Models/Poi.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasMany;

class Poi extends Model
{
    use HasFactory;

    public $timestamps = false;
    protected $fillable = ['name', 'longitude', 'latitude', 'modifier'];

    public function art() :HasMany{
        return $this->hasMany(Art::class);
    }
}
